samples/Azure.ResourceManager.Sample/Generated/Models/VirtualMachineScaleSetExtensionUpdate.Serialization.cs

```diff
@@ -28,10 +28,10 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
                 writer.WritePropertyName("publisher");
                 writer.WriteStringValue(Publisher);
             }
-            if (Optional.IsDefined(TypePropertiesType))
+            if (Optional.IsDefined(TypeAutoGenerated))
```

Comment:
This change is from autorest.core update

Reply:
Is it not a breaking change to rename a property?

Reply:
Yes, it is. @m-nash, this change has affected only mgmt, do you know anything about it?

Reply:
Fixed